benchmark_db_terraform/modules/benchmark_db/service/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM fedora:36
```

Comment:
It might be worth bumping it to `38`. Or using an Ubuntu LTS. While I do like the Fedora Project, its releases don't have long-term support.